Protocols/DeckofCards.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  DeckofCards.swift
+//  Protocols
+//
+//  Created by Felipe Gil on 2021-09-24.
+//
+
+import Foundation
+
+
+
+class DeckofCards: CardsDataSource {
```

Comment:
Your implementation of the CardsDataSource is fine, but there is room for improvement. Firstly I'd like to ask you what's the reason for making a class here. Also, could you create another data source, now for Spanish cards? And the finally, the most important: do you play Truco?

Reply:
Thinking about it, there is no reason at all. The only "inheritance-ish" that I'm using is from protocols, in fact, more like conformance than inheritance. and I need it by value not reference. 
So I'm updating this to be a Struct.